about.html

```diff
@@ -0,0 +1,126 @@
+        <h1>Tomorrowland <br> world tour concert 2022</h1>
+        <div class="middle-text">
+          <p>THE PEOPLE OF TOMORROW<br>
+            We believe...<br>
+            in a commitment to create a reality that relates positively with Mother Nature and contributes to the well-being of the next generations. Let’s engage with each other and do something good today, for which we will be grateful for tomorrow.</p>
+        </div>
+      </div>
+      <div class="about-date-place">
+        <h2>Please Contact us per Email for any further question about Tomorrowland world tour concert 2022</h2>
+        <h3>Tomorrowland2022@cckorea.org</h3>
+      </div>
+    </section>
+    <section class="logo-section">
+      <div class="about-featured-head"> 
+        <h2>Tomorrowland world tour concert 2022 Logo</h2>
+        <hr class="underline">
+      </div>
+      <div class="logo-disc">
+        <h2>Please Contact us per Email for any further question about Tomorrowland world tour concert 2022</h2>
+      </div>
+      <div class="logo-img">
+        <img src="images/footer-img.png" alt="Concert Logo">
+      </div>
+      <hr>
+      <div class="about-featured-head"> 
+        <h2>See the past Tomorrowland world tour concert</h2>
+        <hr class="underline">
+      </div>
+      <div class="logo-disc">
+        <h2>Please Contact us per Email for any further question about Tomorrowland world tour concert 2022</h2>
+      </div>
+      <div class="past-wrapper">
+        <div class="past-container">
+          <div class="past-image1">
+            <div class="past-image-wrapper">
+            </div>
+          </div>
+          <div class="past-image-disc">
+            <h2>2019</h2>
+            <h3>CC Globbal Summit 2020</h3>
+          </div>
+        </div>
+        <div class="past-container">
+          <div class="past-image2">
+            <div class="past-image-wrapper">
+            </div>
+          </div>
+          <div class="past-image-disc">
+            <h2>2020</h2>
+            <h3>CC Globbal Summit 2020</h3>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
+  <section class="about-partner-section">
+    <div class="partner-section-wrapper">
+      <div class="featured-head"> 
+        <h2>Partners</h2>
+        <hr class="about-underline-partner">
+      </div>
+      <div class="partners-img">
+        <img class="sandbox-img" src="images/nike.png" alt="Logo">
+        <img class="viking-img" src="images/airbnb.png" alt="Logo">
+        <img class="sandbox-img" src="images/nike.png" alt="Logo">
+        <img class="viking-img" src="images/airbnb.png" alt="Logo">
+        <img class="sandbox-img" src="images/nike.png" alt="Logo">
+      </div>
+      <hr>
+    </div>
+  </section>
+
+  <footer class="footer-section">
+    <div class="footer-wrappter">
+        <img src="images/footer-img.png" alt="creative-common">
+        <p>2022 Tomorrowland world tour Rights Reserved</p>
+    </div>
+  </footer>
```

Comment:
- [ ] Desktop: About page
- Please the footer on this page has a dark background. Kindly follow 👉  [the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
- See your design:
![Capture](https://user-images.githubusercontent.com/63972670/180995336-a4ea4fe9-55b7-40fd-9388-09c914f4b28c.PNG)